.travis.yml

```diff
@@ -0,0 +1,34 @@
+version: ~> 1.0
+language: scala
+
+env:
+  global:
+    - TRAVIS_JDK=11
+
+before_install:
+  - curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
```

Comment:
IIRC, `git.io/jabba` redirects to a `master` version of the script. We've moved to a stable version of the script in [several places](https://github.com/lagom/lagom/blob/master/.travis.yml#L26).